test/parallel/test-runner-mock-timers-date.js

```diff
@@ -0,0 +1,122 @@
+'use strict';
+process.env.NODE_TEST_KNOWN_GLOBALS = 0;
+require('../common');
+
+const assert = require('node:assert');
+const { it, describe } = require('node:test');
+
+describe('Mock Timers Date Test Suite', () => {
+  describe('Date Suite', () => {
```

Comment:
I don't think we need double describes for this, I think just the inner one will do

Reply:
Fixed